color-identifiers-mode.el

```diff
@@ -101,9 +102,10 @@ error colors etc."
   "Alist of major modes and the ways to distinguish identifiers in those modes.
 The value of each cons cell provides four constraints for finding identifiers.
 A word must match all four constraints to be colored as an identifier.  The
-value has the form (IDENTIFIER-CONTEXT-RE IDENTIFIER-RE IDENTIFIER-FACES
+value has the form (MAJOR-MODE IDENTIFIER-CONTEXT-RE IDENTIFIER-RE IDENTIFIER-FACES
 IDENTIFIER-EXCLUSION-RE).
 
+MAJOR-MODE is the name of mode in which this cons cell should be used
```

Comment:
The cons cell is just the data structure we use to hold each rule, so it may be clearer to refer to it as a rule instead.

```suggestion
MAJOR-MODE is the name of the mode in which this rule should be used.
```

Reply:
Thanks, done!